tasks/setup.yml

```diff
@@ -3,7 +3,8 @@
 
 # We are using the raw module here because of passwords generated with /dev/urandom :-(
 - name: Setup Tuleap
-  raw: /usr/share/tuleap-install/setup.sh --sys-default-domain="{{ tuleap_domain }}" --sys-org-name="{{ tuleap_org_name }}" --sys-long-org-name="{{ tuleap_org_name }}"
-  notify: 
+  raw: /usr/share/tuleap/tools/setup.el7.sh --configure --server-name={{ ansible_fqdn }} --mysql-server=localhost --assumeyes
```

Comment:
setup.el7.sh will not work on CentOS 6, I'm not sure we want to drop the support for it now, so you should probably allow the old setup.sh to be ran.

It would also be better to keep the use of tuleap_domain as there are multiple case where this is not the same as the full fqdn of the server (tuleap_domain could be set by default to ansible_fqdn).

Reply:
Well, this PR deals with centos7. The install procedure changes between the version 6 and 7.
What about manage centos6 in a dedicated branch? Maybe, you can create a branch 'centos6' and the master branch becomes the recommended centos7. There is no risk because I added a check on the OS major version in main.yml.

No problem to keep the parameter `sys-default-domain`. I removed it because it is no more used in the [official documentation](https://docs.tuleap.org/installation-guide/full-installation.html#setup). Actually, I tried to follow the official documentation. As said in my first comment, the only gap with the documentation concerns the mysql password.